src/Microsoft.DotNet.Interactive.Formatting/DefaultJsonFormatterSet.cs

```diff
@@ -1,17 +1,23 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using System;
-using System.Collections.Concurrent;
-using Newtonsoft.Json.Linq;
+using System.Text.Encodings.Web;
+using System.Text.Json;
+using System.Text.Unicode;
 
 namespace Microsoft.DotNet.Interactive.Formatting
 {
     internal class DefaultJsonFormatterSet 
     {
-        static internal readonly ITypeFormatter[] DefaultFormatters =
+        internal static readonly ITypeFormatter[] DefaultFormatters =
             new ITypeFormatter[]
             {
+                new JsonFormatter<string>((context, s, writer) =>
+                {
+                    var data = JsonSerializer.Serialize(s, JsonFormatter.SerializerSettings);
+                    writer.Write(data);
+                    return true;
```

Comment:
It looks like this logic is repeated insie the `JsonFormatter` class. Is any change actually needed in this file?